src/main/java/org/broadinstitute/barclay/argparser/CommandLinePluginDescriptor.java

```diff
@@ -161,7 +161,7 @@ public abstract Object getInstance(Class<?> pluggableClass)
      * @return the default plugins used for this instance of this descriptor as a list of Object. Used for
      * help/doc generation.
      */
-    public abstract List<Object> getDefaultInstances();
+    public abstract List<? extends T> getDefaultInstances();
 
     /**
      * @return an ordered List of actual plugin instances that have been specified on the command
```

Comment:
We should audit each place where a wildcard or Object appears, including the Class<> references, and tighten the bounds if appropriate. But before we merge anything I'd want to verify it by building GATK with the changes to make sure the effects on the GATK plugin descriptor are reasonable, and to get confidence that we haven't been overly restrictive.